### PR TITLE
Fix apply on Ansible 2.9

### PR DESCRIPTION
**Pull request by gravesm**
##### SUMMARY

For some reason the apply function can't be correctly imported in
Ansible 2.9. This just renames it to get it to import. I've also added
molecule testing on multiple Ansible versions.

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/module_utils/k8sdynamicclient.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 from kubernetes.dynamic import DynamicClient
 
-from ansible_collections.kubernetes.core.plugins.module_utils.apply import apply
+from ansible_collections.kubernetes.core.plugins.module_utils.apply import k8s_apply
```
Comment (abikouo) [inline plugins/module_utils/k8sdynamicclient.py:22]: ``import apply as k8s_apply`` why not ?

Comment (gravesm) [reply plugins/module_utils/k8sdynamicclient.py:22]: This is broken as well. The only way I can get this to work is to just rename the function.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -75,6 +75,7 @@ jobs:
     strategy:
       matrix:
         python_version: ['3.7']
+        ansible_version: ['==2.9.*', '']
```
Comment (abikouo) [inline .github/workflows/ci.yml:78]:
```suggestion
        ansible_version: ['==2.9.*', '==2.10.*', '==2.11.*', '']
```

File: plugins/action/k8s_service.py [hunk 3]
```diff
@@ -0,0 +1 @@
+k8s_info.py
```
Comment (Akasurde) [inline plugins/action/k8s_service.py:1]: I didn't realize runtime.yml does not work on 2.9. @gravesm Thanks for catching this.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (gravesm) [reply .github/workflows/ci.yml:78]: This won't work because there is no 2.11 release of ansible.

Comment (abikouo) [reply .github/workflows/ci.yml:78]: so at least 2.10 in order to validate on different version

**Review by abikouo — APPROVED** [review]

**Review by goneri — APPROVED** [review]

**Pull request merged**